x/concentrated-liquidity/tick_test.go

```diff
@@ -601,6 +608,132 @@ func (s *KeeperTestSuite) TestCrossTick() {
 	}
 }
 
+func (s *KeeperTestSuite) TestGetTickLiquidityForRange() {
+	defaultTick := withPoolId(defaultTick, defaultPoolId)
+
+	tests := []struct {
+		name        string
+		presetTicks []genesis.FullTick
+
+		expectedLiquidityDepthForRange []query.LiquidityDepthWithRange
+	}{
+		{
```

Comment:
Can we add a test case with 2 full range and 2 narrow range positions?

I don't think there is one where we have both full range and narrow range

Reply:
added :)